docs/docs/deprecatedMetrics.md

```diff
@@ -0,0 +1,133 @@
+
+We have statistics about read and write requests for each repository. To access them, substitute `*` with the name of a repository:
+
+`service.mesosphere.marathon.state.*.read-request-time.count` - The number of read requests.
+
+`service.mesosphere.marathon.state.*.read-request-time.mean` - The exponential weighted average of the read request times.
+
+`service.mesosphere.marathon.state.*.write-request-time.count` - The number of write requests.
+
+`service.mesosphere.marathon.state.*.write-request-time.mean` - The [exponential weighted average](https://dropwizard.github.io/metrics/3.1.0/manual/core/#exponentially-decaying-reservoirs) of the write request times.
+
+**Note:** Many of the repository metrics were not measured correctly prior to <span class="label label-default">v0.15</span>.
+
+### Requests
+
+`org.eclipse.jetty.servlet.ServletContextHandler.dispatches` (timer) - The
+number of HTTP requests received by Marathon is available under `.count`.
+There are more metrics around HTTP requests under the
+`org.eclipse.jetty.servlet.ServletContextHandler` prefix.
+For more information, consult [the code](https://github.com/dropwizard/metrics/blob/796663609f310888240cc8afb58f75396f8391d2/metrics-jetty9/src/main/java/io/dropwizard/metrics/jetty9/InstrumentedHandler.java#L41-L42).
+
+### JVM
+
+`jvm.threads.count` (meter) - The total number of threads. This number should be below 500.
+
+`jvm.memory.total.used` (meter) - The total number of bytes used by the Marathon JVM.
+
+## Instrumented method calls
+
+These metrics are created automatically by instrumenting certain classes in our code base.
+
+You can disable these instrumented metrics with `--disable_metrics`. This flag will only disable this code instrumentation, not all metrics.
+
+These timers can be very valuable in diagnosing problems, but they require detailed knowledge of the inner workings of Marathon. They can also degrade performance noticeably.
+
+Since these metric names directly correspond to class and method names in our code base,
+expect the names of these metrics to change if the affected code changes.
+
+## Potential pitfalls
+
+### Derived metrics (mean, p99, ...)
+
+Our metrics library calculates derived metrics like "mean" and "p99" using a sliding average window algorithm. This means that every time you fetch the `/metrics` endpoint you are looking at the average of the last N seconds. By default the length of the window is 30 seconds, but this can be configured with the `--metrics_averaging_window` flag.
+
+For getting the most accurate results it is recommended to configure your polling interval to the size of the sliding average window.
+
+### Statsd and derived statistics
+
+Statsd typically creates derived statistics (mean, p99) from mean values Marathon reports. Our metrics package also reports derived statistics. To avoid accidentally aggregating statistics multiple times, be sure you know where you are reporting and computing mean values.
```

Comment:
these docs are killer!   👏

Reply:
I think I should generalize it and add it to the new metrics doc. :)